script.js

```diff
@@ -151,31 +154,21 @@ class Player{
             }
 
             if (this.hand[i]  == 'A'){
-                if (this.player_score > 10){
-                this.player_score += 1;
+                if (this.score > 10){
+                this.score += 1;
                 }
                 else{
-                this.player_score += 11;
+                this.score += 11;
                 }
                 continue;
             }
 
             // handle non-jqka cards
-            this.player_score += this.hand[i];
+            return this.score += this.hand[i];
```

Comment:
remember that `return` immediately exits the function and returns whatever the value being returned is!

i think you want to `return this.score` outside the for loop

Reply:
fixed... that makes more sense